--- linux-gpib-kernel-4.3.3.org/drivers/gpib/Makefile	2020-05-09 10:08:29.000000000 +0200
+++ linux-gpib-kernel-4.3.3/drivers/gpib/Makefile	2021-02-25 21:49:38.659087538 +0100
@@ -2,6 +2,9 @@ ifneq ($(LINUXINCLUDE),)
 	LINUXINCLUDE := -I$(GPIB_TOP_DIR)/compat/include $(LINUXINCLUDE)
 endif
 
+KERNEL_GE_5_10 = $(shell expr \( $(VERSION) '>' "5" \) \| \
+ \( \( $(VERSION) '=' "5" \) \& \( $(PATCHLEVEL) '>=' 10 \) \) )
+
 subdir-ccflags-y := -I$(src)/include \
 			-DHAVE_DEV_OF_NODE=$(HAVE_DEV_OF_NODE) \
 			-DGPIB_CONFIG_PCMCIA=$(GPIB_CONFIG_PCMCIA) \
@@ -19,7 +22,9 @@ obj-y += gpio/
 obj-y += hp_82335/
 obj-y += hp_82341/
 obj-y += ines/
-obj-$(CONFIG_USB) += lpvo_usb_gpib/
+ifeq (${KERNEL_GE_5_10},0)
+	obj-$(CONFIG_USB) += lpvo_usb_gpib/
+endif
 obj-y += nec7210/
 obj-$(CONFIG_USB) += ni_usb/
 ifeq ($(CONFIG_GPIB_ISA),yes)
