--- linux-gpib-kernel-4.2.0/drivers/gpib/sys/osfuncs.c~	2018-11-07 15:58:07.000000000 +0100
+++ linux-gpib-kernel-4.2.0/drivers/gpib/sys/osfuncs.c	2019-03-10 18:51:26.364983014 +0100
@@ -551,7 +551,11 @@
 	remain = read_cmd.requested_transfer_count - read_cmd.completed_transfer_count;
 
 	/* Check write access to buffer */
+#if LINUX_VERSION_CODE < KERNEL_VERSION(5,0,0)
 	if(!access_ok(VERIFY_WRITE, userbuf, remain))
+#else
+	if(!access_ok(userbuf, remain))
+#endif
 		return -EFAULT;
 
 	smp_mb__before_atomic();
@@ -626,7 +630,11 @@
 	remain = cmd.requested_transfer_count - cmd.completed_transfer_count;
 
 	/* Check read access to buffer */
+#if LINUX_VERSION_CODE < KERNEL_VERSION(5,0,0)
 	if(!access_ok(VERIFY_READ, userbuf, remain))
+#else
+	if(!access_ok(userbuf, remain))
+#endif
 		return -EFAULT;
 
 	/* Write buffer loads till we empty the user supplied buffer.
@@ -705,7 +713,11 @@
 	remain = write_cmd.requested_transfer_count - write_cmd.completed_transfer_count;
 
 	/* Check read access to buffer */
+#if LINUX_VERSION_CODE < KERNEL_VERSION(5,0,0)
 	if(!access_ok(VERIFY_READ, userbuf, remain))
+#else
+	if(!access_ok(userbuf, remain))
+#endif
 		return -EFAULT;
 
 	smp_mb__before_atomic();
