--- linux-gpib-4.0.3/drivers/gpib/include/gpib_types.h.orig	2016-07-02 16:36:26.318566419 +0200
+++ linux-gpib-4.0.3/drivers/gpib/include/gpib_types.h	2016-07-02 16:47:34.328280834 +0200
@@ -24,7 +24,7 @@
  * and generic interface provided by gpib-common.
  * This really should be in a different header file.
  */
-#include "gpib/gpib_user.h"
+#include "gpib_user.h"
 #include <asm/atomic.h>
 #include <linux/mutex.h>
 #include <linux/wait.h>
--- linux-gpib-3.2.09/drivers/gpib/include/gpibP.h.orig	2007-12-06 00:42:20.000000000 +0000
+++ linux-gpib-3.2.09/drivers/gpib/include/gpibP.h	2007-12-06 00:47:22.000000000 +0000
@@ -20,7 +20,7 @@
 
 #include <linux/types.h>
 
-#include "gpib/gpib_user.h"
+#include "gpib_user.h"
 #include "gpib_types.h"
 #include "gpib_proto.h"
 #include "gpib_ioctl.h"
