--- linux-gpib-user-4.3.0/usb/agilent_82357a/Makefile.am~	2019-08-23 10:30:11.000000000 +0200
+++ linux-gpib-user-4.3.0/usb/agilent_82357a/Makefile.am	2019-09-18 17:53:41.226519040 +0200
@@ -10,7 +10,7 @@
 EXTRA_DIST = 99-agilent_82357a.rules.in
 CLEANFILES = 99-agilent_82357a.rules
 
-USB_FIRMWARE_DIR=$(datadir)/usb
+USB_FIRMWARE_DIR=/lib/firmware
 UDEV_RULES_DIR=$(sysconfdir)/udev/rules.d
 
 do_subst = $(SED) -e 's,[@]sbindir[@],$(sbindir),g'\
--- linux-gpib-user-4.3.0/usb/ni_usb_gpib/Makefile.am~	2019-08-23 10:30:11.000000000 +0200
+++ linux-gpib-user-4.3.0/usb/ni_usb_gpib/Makefile.am	2019-09-18 17:54:07.033184839 +0200
@@ -10,7 +10,7 @@
 EXTRA_DIST = 99-ni_usb_gpib.rules.in
 CLEANFILES = 99-ni_usb_gpib.rules
 
-USB_FIRMWARE_DIR=$(datadir)/usb
+USB_FIRMWARE_DIR=/lib/firmware
 UDEV_RULES_DIR=$(sysconfdir)/udev/rules.d
 
 do_subst = $(SED) -e 's,[@]sbindir[@],$(sbindir),g'\
